tango_simlib/utilities/fandango_json_parser.py

```diff
@@ -45,7 +49,7 @@ def parse(self, json_file):
         with open(json_file) as dev_data_file:
             device_data = json_load_byteified(dev_data_file)
 
-        for data_component, elements in device_data.items():
+        for data_component, elements in list(device_data.items()):
```

Comment:
No need to list the dict items.

```suggestion
        for data_component, elements in device_data.items():
```